BattleNetwork/bnPackageManager.h

```diff
@@ -161,6 +161,9 @@ stx::result_t<bool> PackageManager<MetaClass>::LoadPackageFromDisk(const std::st
 
     auto packageClass = this->CreatePackage<ScriptedDataType>(std::ref(state));
 
+    // run script includes
+    state["do_includes"]();
```

Comment:
does `include(...)` work in the global scope? Do we have to have this function in the script too?

Reply:
Not without some reshuffling.
The code for the script is run before `_modpath` is actually assigned (from the `safe_script_file()` call in `ScriptResourceManager::LoadScript`), so having it in global scope will cause it to retrieve `nil` for a relative path.

Will take a bit this afternoon to see I can get it to be able to invoke it in the global scope without causing an error.

Reply:
Changed it so that `include()` is now invocable from global state.
Shuffled it around so `_modpath` is exposed to the file before any of the functions are run, similar to how other parts of the API are loaded in, now it works just peachy.

Reply:
A+ job!